Tests/CacheAdvanceTests/CacheAdvanceTests.swift

```diff
@@ -461,6 +470,77 @@ final class CacheAdvanceTests: XCTestCase {
         }
     }
 
+    // MARK: Performance Tests
+
+    func test_performance_createCacheAndAppendSingleMessage() throws {
+        measure {
+            // Delete any existing cache.
+            FileManager.default.createFile(atPath: testFileLocation.path, contents: nil, attributes: nil)
+
+            guard let sut = try? createCache(maximumByes: 100, overwritesOldMessages: true, zeroOutExistingFile: false) else {
```

Comment:
We need it because sometimes we're trying to create a new cache instance that reads from a file created by another cache. In those instances, we need to set `zeroOutExistingFile` to `false`.